build-system/tasks/codecov-upload.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const colors = require('ansi-colors');
+const fs = require('fs-extra');
+const log = require('fancy-log');
+const {getStdout} = require('../exec');
+const {isTravisBuild, travisPullRequestSha} = require('../travis');
+const {shortSha} = require('../git');
+
+const {green, yellow, cyan} = colors;
+
+/**
+ * Uploads a single report
+ * @param {string} file
+ * @param {string} flags
+ */
+function uploadReport(file, flags) {
+  const codecovExecutable = './node_modules/.bin/codecov';
+  const codecovCmd = `${codecovExecutable} --file=${file} --flags=${flags}`;
```

Comment:
Could/should this be replaced with `npx codecov`?

Reply:
Tried that, didn't work.

- The [official docs](https://www.npmjs.com/package/codecov#usage) recommend invoking `./node_modules/.bin/codecov`.
- There's an open issue about `npx codecov` not being reliable: https://github.com/codecov/codecov-node/issues/112